### ISSUE TITLE
Top menu bar too similar to page

### ISSUE THREAD

**Issue by Marcus-Rosti** [thecourseforum/theCourseForum#141]
The material redesign looks **amazing**. But there are still some issues I think.  The menu bar on the left might look better and follow the design better if it were like the floating plus button on gmail or other material apps.

And also, the menu bar on the top looks far to similar to the cards.  It ruins the paper conceit since they just run together.  You could maybe keep it white, but you'd need to give it a shadow over the internal card.  

Other than that I don't have any problems with the look! #amazing

![screen shot 2015-07-29 at 10 31 57 am](https://cloud.githubusercontent.com/assets/8094255/8960363/88f7c6b4-35dd-11e5-8a60-2692dd01b646.png)

**Comment by keenadams** [comment]
Yes! The plan is definitely to make the side bar floating and collapsible like the youtube one (and I guess a lot of the other google apps?). We just didn't think we'd have time to do that by august 1st (we'd probably have to move the search function and other things), but it's the next thing I want to do. 

I'll take a look at fixing the header. Everything is a little trickier because it's all built on top of the blue background instead of the gray one.
